src/pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -308,3 +310,23 @@ def _check_datamodule_checkpoint_hooks(trainer: "pl.Trainer") -> None:
             "`LightningDataModule.on_load_checkpoint` was deprecated in"
             " v1.6 and will be removed in v1.8. Use `load_state_dict` instead."
         )
+
+
+def _check_dataloader_none(stage: str, **dataloader_args: Any) -> None:
+    for arg_name, value in dataloader_args.items():
+        dataloader_method = arg_name if not arg_name.endswith("s") else arg_name[:-1]
```

Comment:
If you pass the `RunningStage` object instead of the value, you can use `stage.dataloader_prefix` to construct the method name